srml/support/procedural/src/storage/transformation.rs

```diff
@@ -603,6 +604,11 @@ fn store_functions_to_metadata (
 				#scrate::storage::generator::StorageFunctionModifier::Optional
 			}
 		};
+		let default = default_value.inner.get(0).as_ref().map(|d| &d.expr).map(|d| {
+			quote!( #d )
+		})
+			.map(|q|q.to_string())
+			.unwrap_or_else(|| "Default::default()".to_string());
```

Comment:
I would say, that we make `default` an Option. If no default is given, we store `None` and this implies `Default::default()`. This will also decrease the size of the metadata.

Reply:
No, the field should be an `Option<&'static str>`.